mathesar_ui/src/stores/table-data/columns.ts

```diff
@@ -41,6 +46,20 @@ function preprocessColumns(response?: Column[]): Column[] {
   }) || [];
 }
 
+function api(url: string) {
+  return {
+    get() {
+      return getAPI<PaginatedResponse<Column>>(`${url}?limit=500`);
+    },
+    add(columnDetails: Partial<Column>) {
+      return postAPI<Partial<Column>>(url, columnDetails);
+    },
+    remove(index: Column['index']) {
+      return deleteAPI(`${url}${index}`);
```

Comment:
We should probably end all urls with a trailing slash.

The server seems to do a redirect by adding a trailing slash, if we didn't specify any. We can avoid a roundtrip by adding it here.

Reply:
Addressed in 7d3ba58d38a291cabed95322d59bbd8e26532c8b